ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/MyJobsFilter.tsx

```diff
@@ -2,11 +2,11 @@ import React from 'react';
 import { Switch } from '@patternfly/react-core';
 
 export type MyJobsFilterProps = {
-    showOnlyMyJobs: boolean;
+    viewOnlyMyJobs: boolean;
```

Comment:
We use the term `view` in the UI instead of `show` so I'm refactoring this to be consistent.

Reply:
I also like the `should` prefix for booleans that drive behavior.

Reply:
That's a good point. I didn't think about it that deeply when I made that change. At first, I thought it seemed odd to have the term `showOnlyMyJobs` when the label would say "View only my jobs". But considering the nature of the variable (storing a boolean value), maybe keeping the pattern of using `has` or `is` makes more sense. Like calling it `isViewingOnlyMyJobs`. I haven't seen `should` used anywhere else. Is that also a common pattern?

Reply:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Components/CompoundSearchFilter/components/CompoundSearchFilter.tsx#L102-L106

Reply:
Wow! And I wrote it haha. I'm ok with either of the options. I was thinking of sticking to `isViewingOnlyMyJobs` in this case